app/views/contexts/list_context.py

```diff
@@ -0,0 +1,84 @@
+from functools import partial
+
+from flask import url_for
+from flask_babel import lazy_gettext
+
+from . import Context
+
+
+class ListContext(Context):
+    def __call__(
+        self,
+        summary_definition,
+        for_list,
+        return_to=None,
+        editable=True,
+        edit_block_id=None,
+        remove_block_id=None,
+    ):
+        list_items = list(
+            self._build_list_items_context(
+                summary_definition,
+                for_list,
+                return_to=return_to,
+                edit_block_id=edit_block_id,
+                remove_block_id=remove_block_id,
+            )
+        )
+        return {"list": {"list_items": list_items, "editable": editable}}
```

Comment:
It looks like `editable` is passed to call but not propagated on to `_build_list_items_context` so is just being used to set the `editable` property in the context. We should consider removing it from `__call__` and setting it in the context if either `edit_block_id` or `remove_block_id` are passed.